src/bin/nydusd/main.rs

```diff
@@ -20,17 +20,19 @@ use std::convert::TryInto;
 use std::fs::File;
 use std::io::{Read, Result};
 use std::ops::DerefMut;
+use std::sync::Mutex;
```

Comment:
Line 23 and line 26 seems redundant?

Reply:
Mutex is required by lazy_inited global variable,(line 62). PS: Maybe not a good choice, changed later.
Line 26 required by grpc server.

Reply:
I mean you removed `Mutex` at line 26 and add it back at line 23. So the change is unnecessary:)